### PR TITLE
Updating building vms guide for vCD 9.1

### PR DESCRIPTION
**Pull request by suehighmoor**
**Main changes**
Build VMs (vCD 9.1) -- new article
Build VMs -- updated to reference vCD 8.20

**Additional changes**
Portal Getting Started Guide -- separated out steps for memorable word and 2FA; removed incorrect link to Elevated Portal
Portal notifications -- fixed incorrect title in metadata
VMware Getting Started Guide -- removed incorrect link to Elevated Portal

### CODE REVIEW

File: articles/portal/ptl-gs.md [hunk 1]
```diff
@@ -51,26 +51,31 @@ The first thing you need to do is log in.
 1. Go to the appropriate URL, depending on the security domain in which your service is located:
 
     - **Assured OFFICIAL:** <https://portal.ukcloud.com>
-    - **Elevated OFFICIAL:** <https://portal.ukcloud.gsi.gov.uk>
+    - **Elevated OFFICIAL:** Contact your Portal administrator or Customer Success Manager (CSM)
```
Comment (BaileyLawson) [inline articles/portal/ptl-gs.md]: Full stop at the end

File: articles/vmware/vmw-gs.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ The first thing you need to do to get started with UKCloud for VMware is to log
 1. Go to the appropriate URL, depending on the security domain in which your service is located:
 
     - **Assured OFFICIAL:** <https://portal.ukcloud.com>
-    - **Elevated OFFICIAL:** <https://portal.ukcloud.gsi.gov.uk>
+    - **Elevated OFFICIAL:** Contact your Portal administrator or Customer Success Manager (CSM)
```
Comment (BaileyLawson) [inline articles/vmware/vmw-gs.md]: Full stop at the end

File: articles/portal/ptl-gs.md (hunk 1 above)
Comment (suehighmoor) [reply articles/portal/ptl-gs.md]: No need for bullet points to end in full stops if they are single, short sentences. As long as all bullets within a list are consistent.

File: articles/vmware/vmw-gs.md (hunk 2 above)
Comment (suehighmoor) [reply articles/vmware/vmw-gs.md]: No need for bullet points to end in full stops if they are single, short sentences. As long as all bullets within a list are consistent.

**Pull request merged**